src/parser/parser_prod.nit

```diff
@@ -7271,6 +7271,48 @@ redef class ANamedargExpr
 		v.enter_visit(_n_expr)
 	end
 end
+redef class ASafeExpr
+	init init_asafeexpr (
+		n_expr: nullable AExpr,
+		n_quest: nullable TQuest
+	)
+	do
+		_n_expr = n_expr.as(not null)
+		n_expr.parent = self
+		_n_quest = n_quest.as(not null)
+		n_quest.parent = self
+	end
+
+	redef fun replace_child(old_child: ANode, new_child: nullable ANode)
+	do
+		if _n_expr == old_child then
+			n_expr = new_child.as(AExpr)
+			return
+		end
+		if _n_quest == old_child then
+			n_quest = new_child.as(TQuest)
+			return
+		end
+	end
+
+	redef fun n_expr=(node)
+	do
+		_n_expr = node
+		node.parent = self
+	end
+	redef fun n_quest=(node)
```

Comment:
Why no new line before but two after?

Reply:
parser_prod.nit is generated. Git metadata is set on this file to disable the diff but github does not care and shows it anyway :(